stage1/modules/figma/README.md

```diff
@@ -17,12 +17,12 @@
 1.  Вам необходимо пройти тест "Figma Test" в RS APP > Auto Test
```

Comment:
> 2. Раздел "Практика"
> Магическая ссылка "@helenakrasnova" без пояснения куда она ведет (наверное это аккаунт человека которому нужно отправить благодарность? :))

remove link because it doesn't lead anywhere